app/assets/javascripts/wca-autocomplete.js.coffee

```diff
@@ -29,7 +29,7 @@ $(document).ready ->
   }
 
   auto.bind "typeahead:selected", (event, object) ->
-    names = object.name.split(" ")
+    names = object.name.replace(/\s+/, '\x01').split('\x01')
```

Comment:
what's `\x01`?

Reply:
How about something like this:
```js
names = object.name.split(" ")
firstName = names[0]
lastName = names.slice(1, names.length).join(" ")
```
I think that's a bit easier to understand and seems to do the right thing.

Reply:
```\x01``` is a hex value which is used as a temporary marker for splitting. It's unlikely that this appears in normal text and I think it's quite common to use this for regular expressions like this one.
But sure, you can do that without a regular expression and I can change the PR if you want me to do that.

Reply:
Sorry, on mobile for the next weeks (🚴), so I didn't take a deeper look into the issue.

> It's unlikely that this appears in normal text 

I'd prefer a solution without such uncertainty, though. Even if the probability is ridiculously small.